tigerpath/majors_and_certificates/majors/COS-BSE_2018.json

```
@@ -188,7 +190,7 @@
           "name": "General",
           "max_counted": 1,
           "min_needed": 2,
-          "explanation": "Any computer science (COS) course at the 300 or 400 level (except independent work) counts as a computer science departmental.\nIn addition, any 300- or 400- level Math or ELE or Physics or ORF course that does not duplicate COS content, Philosophy 312, MAE 345, CHM 303(CHM 303 is now renumbered to CHM 301 as of 8/27/2018), ECO 312, MOL 437/NEU 437, NEU 330, MUS 315, and ECO 326 – Economics of the Internet count as departmentals.  On occasion, special courses with computational content from other departments may fall in to this category.\n\nOne additional semester of independent work may be counted as one of the departmental courses.",
+          "explanation": "<i>Editor's Note: Beginning with the Class of 2023 CHM 301, ECO 312 and Physics courses will no longer count as COS departmentals.</i>\n\nAny computer science (COS) course at the 300 or 400 level (except independent work) counts as a computer science departmental.\nIn addition, any 300- or 400- level Math or ELE or Physics or ORF course that does not duplicate COS content, Philosophy 312, MAE 345, CHM 303(CHM 303 is now renumbered to CHM 301 as of 8/27/2018), ECO 312, MOL 437/NEU 437, NEU 330, MUS 315, and ECO 326 – Economics of the Internet count as departmentals.  On occasion, special courses with computational content from other departments may fall in to this category.",
```

```suggestion
          "explanation": "<i>Note: Beginning with the Class of 2023 CHM 301, ECO 312 and Physics courses will no longer count as COS departmentals.</i>\n\nAny computer science (COS) course at the 300 or 400 level (except independent work) counts as a computer science departmental.\nIn addition, any 300- or 400- level Math or ELE or Physics or ORF course that does not duplicate COS content, Philosophy 312, MAE 345, CHM 303(CHM 303 is now renumbered to CHM 301 as of 8/27/2018), ECO 312, MOL 437/NEU 437, NEU 330, MUS 315, and ECO 326 – Economics of the Internet count as departmentals.  On occasion, special courses with computational content from other departments may fall in to this category.",
```
Same as in the AB version.